sqlalchemy_exasol/requirements.py

```diff
@@ -198,3 +198,33 @@ def standalone_null_binds_whereclause(self):
         WHERE clause, in situations where it has to be typed.
         """
         return exclusions.closed()
+
```

Comment:
I guess this file, was already at this path. However, it actually has to do with tests.

Reply:
This must be part of the dialect itself (at least according to the [documentation](https://github.com/sqlalchemy/sqlalchemy/blob/main/README.dialects.rst#dialect-layout))

Reply:
ok, I thought so also first, because it is somewhat a description of capabilities, was only surprised by the testing imports